### ISSUE TITLE
Make Image Permits less restrictive

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#23807]
### Proposed Objective

Accessibility (WCAG)

### Proposed Priority

Priority 1 - Show Stopper

### dotCMS Version

23.01

### Problem Statement

Image manipulation on the fly is CPU and memory heavy.  In version 23.01, in order prevent runaway CPU/memory usage, we introduced the idea of limiting the ImageFilter to handle a fixed number of permitted requests at any given time - with the intent that it would only affect/limit image manipulation, CPU heavy events.   This works.  By default we limit this to 10 requests, which can be changed by setting `DOT_IMAGE_GENERATION_SIMULTANEOUS_REQUESTS` to a higher number.

But we also cache the output of any given ImageFilter action.  It turns out that requests that could be served with a pre-existing cached result are being run through the 10 requests limit too.  You can see this on say the demo site if you tail the logs and then in another window go and view the content search screen with a ton of thumbnails.  You will see we are hitting the upper limit of the permits.  Another way to hit the upper limit is to run `ab` with a greater concurrency than the set limit, e.g.

```
ab -n 10000 -c 20 http://127.0.0.1:8080/dA/4694d40b-d9be-4e09-b031-64ee3e7c9642/image/270w/50q/diving.jpg
```
Doing this, you will see the warnings in the logs that the "Image permits are exhausted".  This should not be happening.

Now, setting DOT_IMAGE_GENERATION_SIMULTANEOUS_REQUESTS=500 or something will get rid of the message, but it will also defeat the original purpose of only letting a limited number of threads into the ImageFilter at a given time.

We need a way to 
1. serve from cache as speedily as possible and 
3. to limit the number of CPU/Memory intensive image manipulations at any one time.

### Steps to Reproduce

```
ab -n 10000 -c 20 http://127.0.0.1:8080/dA/4694d40b-d9be-4e09-b031-64ee3e7c9642/image/270w/50q/diving.jpg
```
and tail the logs

### Acceptance Criteria

be able to run `ab` without seeing the "Image permits exhausted" messaging.

**Comment by dsilvam** [comment]
This one produced failing tests on Content Resource https://github.com/dotCMS/core/pull/23840. Please check on release branch

**Comment by josemejias07** [comment]
**Approved QA** - Tested on **23.02_20326b3b_SNAPSHOT** // Docker // macOS 13.0 // FF v109.0.1 
![Screenshot 2023-02-02 at 10 43 13 AM](https://user-images.githubusercontent.com/110418590/216387275-00ba3985-14b1-44ba-a259-cbafa1c304a0.png)
